.pre-commit-hooks.yaml

```diff
@@ -8,4 +8,13 @@
   pass_filenames: false
   additional_dependencies: []
   minimum_pre_commit_version: "2.9.2"
-
+- id: uv-lock
+  name: uv-lock
+  description: "Automatically run 'uv lock' on your project dependencies"
+  entry: uv lock
+  language: python
+  files: ^uv.lock$
```

Comment:
We actually still need to check if the `pyproject.toml` (and, I think, `uv.toml`) files changed, because they would change the inputs to the `uv.lock` file.